app/src/main/java/org/mozilla/fenix/settings/creditcards/CreditCardsSettingFragment.kt

```diff
@@ -158,4 +196,54 @@ class CreditCardsSettingFragment : PreferenceFragmentCompat() {
 
         isCreditCardsListLoaded = true
     }
+
+    /**
+     * Show a warning to set up a pin/password when the device is not secured. This is only used
+     * when BiometricPrompt is unavailable on the device.
+     */
+    override fun showPinDialogWarning(context: Context) {
```

Comment:
Going back to what we said in https://github.com/mozilla-mobile/fenix/pull/19548#discussion_r640037358, we probably don't need a kDoc for these override fun as well

Reply:
We are specifying functionality that is specific to the credit card setting fragment. It is overriding an abstract function and should be documented.

Reply:
Cool, that's fine.